### DISCUSSION TITLE
我想使用event.comment.strip()获得加群时输入的答案，报错提示却是comment是个空值

### DISCUSSION THREAD

**Discussion by Ah-Q** [LagrangeDev/Lagrange.Core#217] [Q&A]
`无法触发的群`
![image](https://github.com/KonataDev/Lagrange.Core/assets/80017370/305f6922-a6ea-4f1f-87ad-152f39427184)
`协议端`
![image](https://github.com/KonataDev/Lagrange.Core/assets/80017370/eda11a79-736a-4015-aabf-f539e60ee83c)
`入群问题`
![image](https://github.com/KonataDev/Lagrange.Core/assets/80017370/abc2458d-2902-40c3-a052-668a984960f8)
`离谱的是换个群就能获得消息`
![image](https://github.com/KonataDev/Lagrange.Core/assets/80017370/b69cb22d-92fb-415e-b63d-4a59923f5586)
`不同的是协议端多了一大堆东西`
![image](https://github.com/KonataDev/Lagrange.Core/assets/80017370/db209f37-a82f-4e4f-b2a4-dc13bb9a39f0)

**Comment by ghost** [comment]
@Linwenxuan05 

It seems to be a matter of protocol?

**Comment by ghost** [comment]
无法复现.png

**Reply by Ah-Q** [reply]
佬，能抢救抢救不

**Reply by pk5ls20** [reply]
> 佬，能抢救抢救不

你看下，和 #504 的情况一样吗？

**Comment by YUCLing** [comment]
Possible related to https://github.com/LagrangeDev/Lagrange.Core/issues/504

**Reply by pk5ls20** [reply]
#504 是由于**加群者是风险账号**，从而无法从正常加群申请列表拉取到加群者信息的问题
